### ISSUE TITLE
Snailrace - Multiple Races

### ISSUE THREAD

**Issue by 49Indium** [UQComputingSociety/uqcsbot-discord#106]
Snail race seems to be able to start multiple races at once, leading to weird behavior (see [here](https://discord.com/channels/813324385179271168/836243768411160606/1102497739088084993)).

**Comment by andrewj-brown** [comment]
Unsure if this should be `wontfix` considering snailrace dev has moved to a different repo. Might tag as a potential issue for new contributors to work on, but needs a little bit more explanation of the bug.

**Comment by lcox74** [comment]
Yeah I'm working on the snailrace standalone bot and won't be doing this one. So if someone else wants to try and implement this go for gold.

**Comment by andrewj-brown** [comment]
removing good-first-issue because concurrency is Hard:tm:. noting that snailrace may not be long for this world, anyway.
